{"tests": [{"name": "Check lcs(0)", "max_points": 1, "filename": "test_0", "points": 0, "files": [], "message": "The callable 'lcs' supplied in your code failed with an exception while it was being called by the grading code:Traceback (most recent call last):\n  File \"/grade/run/code_feedback.py\", line 408, in call_user\n    return f(*args, **kwargs)\nTypeError: lcs() missing 1 required positional argument: 'B'\n\n"}], "score": 0.0, "succeeded": true, "gradable": true, "max_points": 1, "images": []}